README.md

```diff
@@ -393,7 +394,17 @@ acquireToken method without activity supports dialog prompt.
 
 ### Encryption
 
-ADAL encrypts the tokens and store in SharedPreferences by default. You can look at the StorageHelper class to see the details. Android introduced AndroidKeyStore for 4.3(API18) secure storage of private keys. ADAL uses that for API18 and above. If you want to use ADAL for lower SDK versions, you need to provide secret key at AuthenticationSettings.INSTANCE.setSecretKey
+ADAL encrypts the tokens and store in SharedPreferences by default. You can look at the StorageHelper class to see the details. Android introduced AndroidKeyStore for 4.3(API18) secure storage of private keys. ADAL uses that for API18 and above. If you want to use ADAL for lower SDK versions, you need to **provide secret key at AuthenticationSettings.INSTANCE.setSecretKey**
+
+```Java
+	SecretKeyFactory keyFactory = SecretKeyFactory
+		.getInstance("PBEWithSHA256And256BitAES-CBC-BC");
+    SecretKey generatedSecretKe = keyFactory.generateSecret(new PBEKeySpec("your_password".toCharArray(),
```

Comment:
We should provide guidelines for key generation, providing code like this would result in people copy pasting the code as it and not changing "your_password" and other params. Also, "your_password" seems confusing.